### ISSUE TITLE
Timesketch doesn't understand when http_response_bytes is "-"

### ISSUE THREAD

**Issue by pyllyukko** [google/timesketch#1935]
**Describe the bug**
When you upload data from Plaso's `apache_access` parser, Timesketch complains and doesn't display log events that have `http_response_bytes` as "-". The following log line can be seen on the worker's log:

```
[400] <mapper_parsing_exception> failed to parse field [http_response_bytes] of type [long] in document with id 'xyz'. Preview of field's value: '-' [illegal_argument_exception/For input string: "-"]
```
[Apache's documentation](https://httpd.apache.org/docs/2.4/mod/mod_log_config.html#logformat) states that: "%b - Size of response in bytes, excluding HTTP headers. In CLF format, i.e. a '-' rather than a 0 when no bytes are sent.".

**To Reproduce**
1. Get Apache logs that have `http_response_bytes` as "-" (see [here](https://github.com/log2timeline/plaso/blob/5198673e2312eac0bbf35fb200cb8bfedea88423/test_data/access.log#L7))
2. Process with Plaso's `apache_access` parser
3. Upload to Timesketch
4. Observe events missing

**Expected behavior**
See all the events in Timesketch.

**Comment by berggren** [comment]
This is because the mapping (ES schema) is set on first event, which is probably a `long`. When a fiueld with `-` is encounter it will try to index as a `long` and fails.

The easy solution is to always treat this attribute as text by updating the plaso.mappings file.

**Comment by berggren** [comment]
Fix is in review in #1941
